internal/configuration/configuration.go

```diff
@@ -32,20 +35,39 @@ const (
 	varKeycloakURL     = "keycloak.url"
 	varMaxRequestRetry = "max.request.retry"
 
+	varOpenShiftApiURL   = "openshift.api.url"
+	varOpenShiftApiToken = "openshift.api.token"
+
 	varIndexPath = "index.path"
 
-	varLocalDevEnv = "local.dev.env"
+	osoClusterConfigFileName    = "oso-clusters.conf"
+	defaultOsoClusterConfigPath = "/etc/fabric8/" + osoClusterConfigFileName
```

Comment:
Should we make this path configurable? (if nothing else, it would simplify testing, so I assume answer is yes:) )

Reply:
Seems this also causes test failures

Reply:
So we most likely need to inject the location as env variable